### PR TITLE
Use the new yolks repo for Pterodactyl images

### PR DESCRIPTION
**Pull request by fabianmakila**
Pterodactyl now provides Java 8, 11 and 16 images by default. The Java images in the `ghcr.io/pterodactyl/yolks` repository are preferred over the ones in `quay.io/parkervcp/pterodactyl-images`.

I've also changed the instructions to better match the current panel behavior. English is not my first language, so I'd be more than happy to get some feedback from a native speaker.

### CODE REVIEW

**Review by sulu5890 — CHANGES_REQUESTED** [review]
Looks fine, however the image will come up very large. Please either upload a smaller image, or set a display size. Here's [how you do that](https://github.com/PaperMC/PaperDocs/blame/master/source/java-update/index.rst#L66).

**Comment by fabianmakila** [comment]
> Looks fine, however the image will come up very large. Please either upload a smaller image, or set a display size. Here's [how you do that](https://github.com/PaperMC/PaperDocs/blame/master/source/java-update/index.rst#L66).

I've now added a width for the image. I can also decrease the original image resolution if the current resolution is too large.

File: source/java-update/index.rst [hunk 1]
```diff
@@ -173,11 +173,12 @@ administrator control panel, go to the *Servers* tab, click on your server
 (this has to be repeated for every server you wish to switch the Java version
 of), and press the *Startup* tab.
 
-Proceed by replacing the text in the *Image* input field under *Docker Container
-Configuration* with the following: ``quay.io/parkervcp/pterodactyl-images:debian_openjdk-16``.
-You can also replace ``16`` with ``11`` for Java 11.
+Proceed by selecting ``ghcr.io/pterodactyl/yolks:java_16`` from the *Image* dropdown under *Docker Container Configuration*.
+If you are running older panel version, manually enter the image url in the custom image field.
```
Comment (NoahvdAa) [inline source/java-update/index.rst:177]:
```suggestion
If you are running an older panel version, manually enter the image url in the custom image field.
```

**Review by NoahvdAa — APPROVED** [review]

File: source/java-update/index.rst [hunk 2]
```diff
@@ -173,11 +173,12 @@ administrator control panel, go to the *Servers* tab, click on your server
 (this has to be repeated for every server you wish to switch the Java version
 of), and press the *Startup* tab.
 
-Proceed by replacing the text in the *Image* input field under *Docker Container
-Configuration* with the following: ``quay.io/parkervcp/pterodactyl-images:debian_openjdk-16``.
-You can also replace ``16`` with ``11`` for Java 11.
+Proceed by selecting ``ghcr.io/pterodactyl/yolks:java_16`` from the *Image* dropdown under *Docker Container Configuration*.
```
Comment (Proximyst) [inline source/java-update/index.rst:176]: Dropdowns? The image must be updated for this.

**Review by Proximyst — CHANGES_REQUESTED** [review]

File: source/java-update/index.rst (hunk 2 above)
Comment (fabianmakila) [reply source/java-update/index.rst:176]: You can see the first commit includes a newer image of the panel which has the dropdown included. Rich diff in GH for some reason doesn't show the updated image.

**Review by sulu5890 — APPROVED** [review]

**Pull request merged**